test/unit/raft.c

```diff
@@ -1893,10 +1893,158 @@ raft_test_split_vote(void)
 	raft_finish_test();
 }
 
+static void
+raft_test_pre_vote(void)
+{
+	raft_start_test(20);
+	struct raft_node node;
+	raft_node_create(&node);
+
+	/* Check leader_idle calculations. */
+	is(raft_node_send_leader(&node,
+				 2 /* Term. */,
+				 2 /* Source. */
+	), 0, "leader notification");
```

Comment:
Vova said the linter only complains here because the first line has an argument after `(`. If the first argument would be on a next line, the linter is supposed to calm down:
```Diff
@@ -1901,9 +1901,10 @@ raft_test_pre_vote(void)
 	raft_node_create(&node);
 
 	/* Check leader_idle calculations. */
-	is(raft_node_send_leader(&node,
-				 2 /* Term. */,
-				 2 /* Source. */
+	is(raft_node_send_leader(
+		&node,
+		2 /* Term. */,
+		2 /* Source. */
 	), 0, "leader notification");
```
Same in the other places. Otherwise the indentation is really way too big.

Reply:
Oh, thanks. Fixed.